main.go

```diff
@@ -268,6 +268,27 @@ func run() int {
 	app.Usage = "Memo Life For You"
 	app.Version = VERSION
 	app.Commands = commands
+	app.Action = func(c *cli.Context) error {
+		args := c.Args()
+		if args.Present() {
+			xcmd := fmt.Sprintf("memo-%s", args.First())
+			xcmdpath, err := exec.LookPath(xcmd)
```

Comment:
OK. I will change code as your suggestion. May I support to override this plugins path by the config? 

example:

```toml
memodir = "path/to/memo"
editor = "vim"
column = 40
selectcmd = "peco"
grepcmd = "grep -nH ${PATTERN} ${FILES}"
assetsdir = "."
# default ` ~/.config/memo/plugins`
pluginsdir = "/path/to/plugins" 
```